climetlab/core/ipython.py

```diff
@@ -17,13 +17,12 @@
 from ..utils import module_loaded
 
 ipython_active = None
-if module_loaded("IPython"):
-    try:
-        from IPython import get_ipython
+try:
+    from IPython import get_ipython as _gipyt
```

Comment:
No good reason to rename "get_ipython" as "_gipyt".

This may be enough:
```python
from IPython import get_ipython
ipython_active = get_ipython()
```

Reply:
the rename code is be delete